packages/babel-node/test/fixtures/cli/--extensions/options.json

```diff
@@ -1,4 +1,4 @@
 {
-  "args": ["foo", "--extensions", ".bar"],
+  "args": ["--extensions", ".bar", "foo"],
```

Comment:
If we exit the `splitArgs` program args scan without seeing `--`, can we scan the `userArgs` and make sure there is no match in node flags? If we have seen such match, we can print a warning message such as:

`--extensions` seems to be a babel-node/node option. Did you mean babel-node --extensions .bar foo? If you intend to pass `--extensions` to your script, please add `--` after `foo`: `babel-node foo -- --extensions. bar`

Of course please improve the wordings here. This message should help people during upgrades from 7 to 8.

Reply:
Will this give rise to false positives? If so I would prefer to keep it as is.

Reply:
This will indeed give false positive: for example, if you have a script that accepts an `--extensions` parameter, it will warn. However, it is easy to suppress the warning (by simply adding `--`), so it could be ok?

We could also remove the warning, for example, after that Babel 8 downloads surpass Babel 7 downloads, so that it's only there for the migration period.

Reply:
That sounds good. :)